### ISSUE TITLE
OCR for digitization

### ISSUE THREAD

**Issue by ekrimmel** [paleo-data/happy-hours#23]
## What is your idea for a happy hour topic?
What's the latest in OCR technology? How can it help us digitize, especially historic labels? Best OCR methods for handwriting, non-English language, etc.

## Who could be involved in leading this happy hour?
Jessica Utrup, possibly with Nelson Rios and/or Rob

## Are there any timing considerations?
_[add text here - e.g. needs to happen before xyz conference, after field season, ASAP, etc.]_

## Next steps
This issue captures a happy hour topic idea. Once you are ready to finalize an idea and put it on the PDWG happy hour schedule, please create a new issue using the [Plan a happy hour](https://github.com/paleo-data/happy-hours/issues/new?assignees=&labels=PDWG&projects=&template=plan-happy-hour.md&title=%5Breplace+this+text+with+the+happy+hour+topic%5D) template. **Then, close this issue.**
